src/els_workspace_symbol_provider.erl

```diff
@@ -30,17 +30,18 @@ handle_request({symbol, Params}, State) ->
 
 -spec modules(binary()) -> [symbol_information()].
 modules(Query) ->
-  {ok, All} = els_dt_document:find_by_kind(module),
+  {ok, All} = els_dt_document_index:find_by_kind(module),
+  Compare   = fun(#{id := X}, #{id := Y}) -> X < Y end,
+  AllSorted = lists:sort(Compare, All),
```

Comment:
This sort add almost nothing to the time it takes to process this request.

Reply:
Did you consider to use an `ordered_set`?

Reply:
I hadn't 😄, but with the current approach the table needs to be a `bag` (ref. `bah` vs. `set` conversation in the other comment).